### PR TITLE
Add check for answers before travel claim submit

### PR DESCRIPTION
**Pull request by wullaski**
## Summary

- Adds check on questions before submitting travel claim and redirects to intro if not

To test:

- Click continue on `/travel-mileage`
- Change address to `/complete`

It should take you back to `/travel-pay`

You should also get the same result if you change the address bar on the `travel-address`,  or `travel-review` pages as well.

## Related issue(s)

- [department-of-veterans-affairs/va.gov-team#79020](https://github.com/department-of-veterans-affairs/va.gov-team/issues/79020)

## Testing done

- Visual
- Unit

## What areas of the site does it impact?

Check-in - travel-claim

## Acceptance criteria

- [ ] It does the check
- [ ] It redirects if appropriate
- [ ] It doesn't break check-in

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/check-in/hooks/usePostTravelClaims.jsx [hunk 1]
```diff
@@ -31,6 +35,14 @@ const usePostTravelClaims = () => {
   );
   useEffect(
     () => {
+      if (
+        !data['travel-vehicle'] ||
```
Comment (brianseek) [inline src/applications/check-in/hooks/usePostTravelClaims.jsx:39]: we should also check to make sure they answered "yes"

**Review by brianseek — COMMENTED** [review]
one little thing, we should also check that they answer yes. You can get to the review page, then select back, then go to /complete and it will go through.

File: src/applications/check-in/travel-claim/pages/complete/Complete.unit.spec.jsx [hunk 2]
```diff
@@ -88,6 +88,28 @@ describe('Check-in experience', () => {
         );
         expect(updateErrorSpy.calledOnce).to.be.true;
       });
+      it('redirects to intro if questions are skipped', () => {
```
Comment (lee-delarm6) [inline src/applications/check-in/travel-claim/pages/complete/Complete.unit.spec.jsx:91]: Do you mean complete page rather than "intro"?

**Review by brianseek — APPROVED** [review]
Looks good to me! :shipit:

**Pull request merged**